webapp/channels/src/components/suggestion/search_channel_with_permissions_provider.test.tsx

```diff
@@ -145,7 +147,7 @@ describe('components/SearchChannelWithPermissionsProvider', () => {
 
         const store = mockStore(state);
 
-        getState.mockImplementation(store.getState);
+        (getState as jest.MockedFunction<() => GlobalState>).mockImplementation(store.getState);
```

Comment:
Same here - we should do this once instead of every time.

Reply:
We might also be able to use `jest.mocked(getState)` or `const getState = jest.mocked(store.getState)` here to avoid having to write out the type for this